src/alignment/Graph.h

```diff
@@ -1,29 +1,57 @@
 #pragma once
 
-#include <fstream>
+#include<fstream>
 #include<string>
 #include<vector>
 #include<array>
 #include<fstream>
+#include "Node.h"
+#include<utility>
 
 class Graph {
-	private:
-		int k;
-		
-		std::vector<std::array<int,3>> listOfEdges;					// vector of edges (as array with size 3)
-		std::vector<std::string> stringListSequence;				// vector of strings for the sequences
-		std::vector<int> numberOfKmer;								// vector amount of k-mer
-		// std::vector<std::vector<int>> adjacentEdges;
-		void calcEdgeList(); 								        // Method: calculate list of edges
-		
-	public:
-	    Graph(int k);											    // constructor
-
-		void readingFastaFiles(std::string nameFile);			    // Method: reading fasta file
-		int getK();									    			// Method: length k-mer
-
-		std::vector<std::string>& getStringListSequence();		    // Method: get StringListSequence
-		std::vector<std::array<int,3>>& getEdgeList();				// Method: get EdgeList
-		std::vector<int> getAdjacentEdge(int sequence, int kNum);	// Method: get adjacent edges from one node
-		const std::vector<int>& getAmountOfKmer();					// Method: get Number of k-mer for all sequences
+    private:
+        // length of any node (kmer)
+        int k;
+
+        // vector for vectors and inside a vector are vectors
+        // for the adjacent nodes of a node
+        // (implicit adjacent edges) 
+        std::vector<std::vector<std::vector<Node> >> listOfEdges;

```

Comment:
An edge is a `vector` of `Node`s?
I suggest you use for now 
```
typedef std::pair <Node, Node> Edge;
```
see http://www.cplusplus.com/reference/utility/pair/pair/.
If necessary, we can then later create a class Edge, e.g.  when edges get properties.